.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+save-prefix false
```

Comment:
True. We use Greenkeeper anyways, so generally we'll get a PR when something is updated and can see the build status. +1 to your change here.